packages/flutter/lib/src/cupertino/radio.dart

```diff
@@ -59,6 +59,8 @@ const List<double> _kDisabledDarkGradientOpacities = <double>[0.08, 0.14];
 
 /// A macOS-style radio button.
 ///
+/// This widget builds a [RawRadio] with a macOS-style UI.
+///
```

Comment:
Nit: These two sentences are somewhat redundant, maybe combine them if you think it would be better.